lib/settings_base.py

```diff
@@ -633,6 +633,10 @@ def JINJA_CONFIG():
             # Collections
             'js/zamboni/collections.js',
 
+            # Unicode: needs to be loaded before collections.js which uses an
```

Comment:
Also the 2nd line doesn't appear to make sense. Could just say "which listens to an event fired in this file" or something.